src/Umbraco.Web.UI.Client/src/views/propertyeditors/rte/rte.prevalues.controller.js

```diff
@@ -22,14 +22,14 @@ angular.module("umbraco").controller("Umbraco.PrevalueEditors.RteController",
 
         tinyMceService.configuration().then(function(config){
             $scope.tinyMceConfig = config;
-
+            //console.log("tinyMceConfig", $scope.tinyMceConfig);
```

Comment:
I think we can extend each command in $scope.tinyMceConfig to have a ```name``` and ```fontIcon``` property as well.

So in the prevalue view instead of using ```<img ng-src="{{cmd.icon}}" />``` we can use ```<i class="{{cmd.fontIcon}}"></i>```

I think it can be compared against the ```frondendCommand``` property.

Eg. most icons use classes like this in the TinyMCE buttons.
```<i class="mce-ico mce-i-bold"></i>```
```<i class="mce-ico mce-i-italic"></i>```
```<i class="mce-ico mce-i-link"></i>```

But some also have custom icons where the frontend command doesn't match the last part in the icon, e.g.
```<i class="mce-ico mce-i-custom icon-picture"></i>``` (Media Picker)
```<i class="mce-ico mce-i-custom icon-settings-alt"></i>``` (Macro)
```<i class="mce-ico mce-i-custom icon-tv"></i>``` (Embed)